src/docfx/build/metadata/ContributionInfo.cs

```diff
@@ -226,23 +231,27 @@ private Repository GetRepository(string path)
                     where file.ContentType == ContentType.Page
                     let fileInRepo = GetRepository(file)
                     where fileInRepo.repo != null
-                    group (file, fileInRepo.pathToRepo) by fileInRepo.repo;
+                    group (file, fileInRepo.pathToRepo)
+                    by fileInRepo.repo;
 
                 foreach (var group in filesByRepo)
                 {
-                    var pathToDocset = group.Select(pair => pair.file.FilePath).ToList();
-                    var pathToRepo = group.Select(pair => pair.pathToRepo).ToList();
                     var repo = group.Key;
                     var repoPath = repo.Path;
-                    var (commitsByFile, allCommits) = GitUtility.GetCommits(repoPath, pathToRepo);
-                    for (var i = 0; i < pathToDocset.Count; i++)
+                    var commitCachePath = Path.Combine(AppData.CacheDir, "commits", HashUtility.GetMd5Hash(repo.Remote));
```

Comment:
>var commitCachePath = Path.Combine(AppData.CacheDir, "commits", HashUtility.GetMd5Hash(repo.Remote)); [](start = 20, length = 101)

can we put all cache file in one place to manage? like commits or github user cache files?